test/transpile/resolve-plugin/cjs.js

```diff
@@ -0,0 +1 @@
+module.exports = true;
```

Comment:
Wondering if we should somehow separate these files from the test files, maybe with a data/ directory? What do you think?